src/site/topic18.rst

```diff
@@ -65,14 +65,45 @@ Call Frame
 Heap
 ====
 
+* Very broadly speaking, the heap can be broken into static space and object space
 
 Static Space
 ------------
 
+* The static space contains one copy of all your code
+    * Classes
+    * Static methods
+    * Static variables
+    * Interfaces
+
 
 Object Space
 ------------
 
+* The object space stores the created objects
+    * The object type
+    * Values of the object's fields
+        * This includes primitive types and references variables to other objects in the heap
+
+* In Java, this is broken down into
+    * Young generation
+    * Old/Tenured generation
```

Comment:
I wouldn't even worry about this for the course. Saying the heap stores objects is sufficient imo.

Reply:
less is more.